deployments/mainnet/weth/migrations/1666906000_configurate_seed_reserves_and_rewards.ts

```diff
@@ -66,19 +66,29 @@ export default migration('1666906000_configurate_seed_reserves_and_rewards', {
         args: [comet.address, COMPAddress],
       },
 
-      // 6. Wrap ETH as WETH and send from Timelock to Comet to seed reserves
+      // 6. Wrap some ETH as WETH
       {
         contract: WETH,
         signature: "deposit()",
         args: [],
-        value: exp(1_000, 18)
-      }, {
+        value: 358052565869157684316n, // 500e18 - current balance
+      },
+
+      // 7. Send all Timelock's WETH to Comet to seed reserves
+      {
         contract: WETH,
         signature: "transfer(address,uint256)",
-        args: [comet.address, exp(1_000, 18)],
+        args: [comet.address, exp(500, 18)],
+      },
+
+      // 8. Transfer COMP
+      {
+        contract: comptrollerV2,
+        signature: '_grantComp(address,uint256)',
+        args: [rewards.address, exp(25_000, 18)],
       },
     ];
-    const description = "# Initialize cWETHv3 on Ethereum" // XXX fill me in
+    const description = "# Initialize cWETHv3 on Ethereum\n\nAs per [the discussion on the forum](https://www.comp.xyz/t/initialize-compound-iii-weth-on-ethereum/3737), Compound Labs has deployed 7 new contracts to mainnet Ethereum in preparation to launch a new cWETHv3 market. Further deployment info can be found on the corresponding [deployment pull request](https://github.com/compound-finance/comet/pull/587\n) to the Comet repository.\n\nThis proposal takes the governance steps recommended and necessary in order to enable the new market. Further detailed information about how the proposal was made can be found on the corresponding [proposal pull request](https://github.com/compound-finance/comet/pull/608).\n\nOnce this proposal is executed, the new cWETHv3 market should be ready for use. Simulations have confirmed this, using the entire [Comet scenario suite](https://github.com/compound-finance/comet/tree/main/scenario).\n\nAlthough the proposal sets the entire configuration in the Configurator, the initial deployment already has most of these same parameters already set. The only actual differences in parameters from the existing deployment implementation are: \n - to raise the supply caps to 80,000 wstETH and 9,000 cbETH, and\n - to set the COMP supply rewards speed to ~38.7 COMP per day, corresponding to the amount cut from cETH(v2)\n\n## Proposal\n\nThe first proposal action sets the cETH(v2) COMP reward speeds to zero in the Comptroller.\n\nThe second action sets the CometFactory for the new Comet instance in the existing Configurator.\n\nThe third action configures the Comet instance in the Configurator.\n\nThe fourth action deploys an instance of the newly configured factory and upgrades the Comet instance to use that implementation.\n\nThe fifth action configures the existing rewards contract for the newly deployed Comet instance.\n\nThe sixth action is to wrap 1,000 ETH from the Timelock, and finally transfer 1,000 WETH to the new Comet instance, in order to seed reserves.\n" // XXX
```

Comment:
link ends with a newline:

```
[deployment pull request](https://github.com/compound-finance/comet/pull/587\n)
```

Reply:
Ok this proposal text is still XXX and going to be updated again (also as TODO on the branch being merged into)